relay-server/src/services/processor/replay.rs

```diff
@@ -128,12 +130,18 @@ fn handle_replay_event_item(
         Ok(replay) => match replay.to_json() {
             Ok(json) => Ok(json.into_bytes().into()),
             Err(error) => {
-                relay_log::error!(error = &error as &dyn Error, "failed to serialize replay");
+                relay_log::error!(
+                    error = &error as &dyn Error,
+                    "failed to serialize replay: {event_id:?}"
```

Comment:
This will put the event_id in a context on the event:

```suggestion
                    event_id = ?event_id,
                    "failed to serialize replay"
```